crowbar_framework/spec/lib/crowbar/lock/local_blocking_spec.rb

```diff
@@ -26,6 +26,11 @@
       end
       expect(lock.locked?).to be false
     end
+  end
+
+  context "when a lock is acquired" do
+
```

Comment:
I agree with the principle. But here it's not worth an extra work of changing existing commits.